sys/fuchsia/fidl_io.txt

```diff
@@ -16,6 +58,22 @@ fuchsia_io_NodeAttributesInLine {
 	modification_time	int64
 } [packed]
 
+fuchsia_io_ObjectInfoInLine [
+	serviceInLine	fuchsia_io_ServiceInLine
+	fileInLine	fuchsia_io_FileObjectInLine
+	directoryInLine	fuchsia_io_DirectoryObjectInLine
+	pipeInLine	fuchsia_io_PipeInLine
+	vmofileInLine	fuchsia_io_VmofileInLine
+	deviceInLine	fuchsia_io_DeviceInLine
+]
```

Comment:
We do not need `[varlen]` attribute here, right? This is statically-sized union?